sdk/include/wdf/kmdf/1.9/wdfstatus.h

```diff
@@ -0,0 +1,188 @@
+/*
```

Comment:
The real source file here should be a .mc file that this file would be generated from.